src/loadConfig.js

```diff
@@ -0,0 +1,33 @@
+const path = require('path')
+const fs = require('fs')
+const YAML = require('yaml')
+
+const CONFIG_FILE_OVERRIDE = 'custom/config.yaml'
+const CONFIG_FILE = 'config.yaml'
+
+function parseJsonOrYaml(filePath) {
+  const content = fs.readFileSync(filePath, 'utf-8')
+  if (filePath.endsWith('.json')) {
+    return JSON.parse(content)
```

Comment:
No need for `JSON`, yaml is actually considered a superset of json, can parse it